chore: rebrand, add lint and CI (#64)
DevExpress/angular-testcafe: README.md

# angular-testcafe [![Tests - E2E](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml/badge.svg)](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml)

A custom Angular builder for [TestCafe](http://devexpress.github.io/testcafe/).
Serves the Angular application, and then runs the TestCafe tests.

# ⚠️**angular-testcafe supersedes @politie/angular-testcafe-builder**⚠️

> Original [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transfered repository and related code, assets and rights to DevExpress. It means that now it is officially maintained and supported under the name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) by original TestCafe authors - the TestCafe team. 

Reviewer: ```suggestion 
[@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transferred the repository, related code, assets, and rights to DevExpress. The TestCafe team maintains and supports the package under the new name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe). 
```

Reviewer 2: Additionally, the @politie/angular-testcafe module in npmjs.com can be set as deprecated: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions and referred to the new version

Author: @timnederhoff, thank you so much for everything that you do for the testing community and TestCafe, in particular 👍 We would really like to notify existing users about the new version via `npm deprecate` if you and your former colleagues don't mind it. Unfortunately, we'll have to bother you again to complete this task.

Reviewer 3: Once you've cut a new release l, I'll mark the existing package as deprecated in npm.

Author: Thank you so much, I just published `angular-testcafe@2.0.6`. After some thinking, I decided to continue original versioning to preserve existing tags and releases in the repository.

Reviewer 3: @AndreyBelym @timnederhoff I've deprecated https://www.npmjs.com/package/@politie/angular-testcafe-builder in the NPM registry.

> If you are still using [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder), it's strongly advised to switch your project to use [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) instead. The development of [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) is discontinued. It won't receive any updates, even for critical bugs and security vulnerabilities.

Reviewer: ```suggestion
If your projects are still using [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder), we recommend that you switch to [angular-testcafe](https://www.npmjs.com/package/angular-testcafe). The development of [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) is discontinued: it won't receive any updates, even for critical bugs and security vulnerabilities.
```

## Install
```bash

> $ npm install --save-dev angular-testcafe

Reviewer 4: If we remove travis CI tests let's remove the badge in the title of README

Reviewer 4: Should we do something with the warning?
![image](https://user-images.githubusercontent.com/12034551/115831100-d33c1580-a419-11eb-87f0-443e846bf212.png)

Author: We can. It's just a bad habit to use `any` everywhere, and we try to not overuse it with this rule in Framework
